src/Microsoft.IdentityModel.Protocols/Configuration/ConfigurationManager.cs

```diff
@@ -222,15 +224,17 @@ public async Task<T> GetConfigurationAsync(CancellationToken cancel)
 
         /// <summary>
         /// Requests that then next call to <see cref="GetConfigurationAsync()"/> obtain new configuration.
-        /// <para>If the last refresh was greater than <see cref="RefreshInterval"/> then the next call to <see cref="GetConfigurationAsync()"/> will retrieve new configuration.</para>
+        /// <para>If the last force refresh was greater than <see cref="RefreshInterval"/> then the next call to <see cref="GetConfigurationAsync()"/> will retrieve new configuration.</para>
         /// <para>If <see cref="RefreshInterval"/> == <see cref="TimeSpan.MaxValue"/> then this method does nothing.</para>
         /// </summary>
         public void RequestRefresh()
         {
             DateTimeOffset now = DateTimeOffset.UtcNow;
-            if (now >= DateTimeUtil.Add(_lastRefresh.UtcDateTime, RefreshInterval))
+            var jitter = new TimeSpan((long)(RefreshInterval.Ticks * _jitterer.Next(0, 100) * 0.01));
```

Comment:
Should we define constants for min and max of the random values and should the max be less than 100? 

With 100, the RefreshInterval may be doubled.

Reply:
The jitter should be applied to _automaticRefreshInterval  when setting with DefaultAutomaticRefreshInterval not necessarily the RequestRefesh interval need not be jittered.
The issue didn't specify, but the idea is in a server farm will not all refresh at the same time.

Reply:
Got it, added change in https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/1674/commits/eeb6eb8daff04b3eb8aa5d2d1be0cc79ad483e1e